### PR TITLE
fix: Rename doc references with graphana to grafana

### PR DESCRIPTION
**Pull request by nextrevision**
Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-rollouts/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [x] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [x] I've signed my commits with [DCO](https://github.com/argoproj/argoproj)
* [x] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [x] My builds are green. Try syncing with master if they are not. 
* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo-rollouts/blob/master/USERS.md).

### CODE REVIEW

File: docs/features/controller-metrics.md [hunk 1]
```diff
@@ -80,4 +80,4 @@ The controller also publishes the following Prometheus metrics to describe the c
 | `workqueue_longest_running_processor_seconds` | How many seconds has the longest running processor for workqueue been running |
 | `workqueue_retries_total`                     | Total number of retries handled by workqueue |
 
-In addition, the Argo Rollouts controllers offers metrics on CPU, memory and file descriptor usage as well as the process start time and current Go processes including memory stats.
\ No newline at end of file
+In addition, the Argo Rollouts controllers offers metrics on CPU, memory and file descriptor usage as well as the process start time and current Go processes including memory stats.
```
Comment (huikang) [inline docs/features/controller-metrics.md:83]: nit: 
- Argo Rollouts controllers offers -> `Argo-rollouts offers`
- current Go processes including memory stats. -> `memory stats of current Go processes`

**Review by huikang — COMMENTED** [review]
Could you rename the title to `fix: ....` to pass the semantic check? Thanks.

File: docs/features/controller-metrics.md (hunk 1 above)
Comment (nextrevision) [reply docs/features/controller-metrics.md]: Added your edits into the branch

**Comment by huikang** [comment]
I think the PR needs to be signed; otherwise LGTM

**Comment by nextrevision** [comment]
Thanks, signed!

**Review by jessesuen — APPROVED** [review]

**Pull request merged**